src/grouping-functions-and-node-capabilities.md

```diff
@@ -10,7 +10,7 @@ This JSON-RPC interface is intended to be implemented on various types of nodes:
 
 - Full nodes, which hold in their database all block headers and bodies, plus the storage of recent blocks.
 - Light clients, which only know the headers of recent blocks.
-- Archive nodes, which hold.
+- Archive nodes, which hold in their database all block headers and bodies, plus the storage of every past block.
```

Comment:
Thanks for noticing this @IkerAlus !

However, I'm afraid that the explanation of "Full nodes" is also off. It should probably say something more or less like:

> - Full nodes, which hold in their database all from the genesis block to the most recent block, plus the bodies and storage of the most recent blocks (typically the latest 256).

Basically, I'm quite certain that it's not true that "full nodes" hold all the bodies from genesis. I'm pretty sure that they only store the headers, which is what it's needed to validate that the state is correct.

Then IMO the "Archive nodes" description should read something more or less like this:

> - Archive nodes, which hold in their database all block headers, bodies, and storage of every past block.

Reply:
yes, I think you are right respect to full nodes description.  

I will change it in this PR too. Thanks!